proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -303,6 +303,20 @@ message MapEntry {
   ImportModuleRef value_import_module_ref = 4;
 }
 
+// OneOf represents a oneof field in a message.
+message OneOf {
+  string name = 1;
+  // the fields that are part of the oneof
+  repeated Field fields = 2;
+}
+
+message MessageField {
+  oneof message_field {
+    Field field = 1;
+    OneOf oneof = 2;
```

Comment:
Why is this embedded within a MessageField, instead of being directly on the Message? Is this to try to maintain ordering of fields for documentation purposes?

Reply:
Yeah, I had this conversation with @SyrieBianco yesterday, and this was the thought behind this. But I believe that is not necessarily a hard requirement for the documentation and we can always handle the sorting on the frontend, so I think I'll be reverting this.